presto-product-tests/src/main/java/io/prestosql/tests/hive/TestHiveBasicTableStatistics.java

```diff
@@ -127,8 +126,10 @@ public void testCreatePartitioned()
                 "WHERE n_nationkey <> 23", tableName));
 
         try {
-            BasicStatistics tableStatistics = getBasicStatisticsForTable(onHive(), tableName);
-            assertThatStatisticsAreNotPresent(tableStatistics);
+            if (getHiveVersionMajor() < 3) {
+                BasicStatistics tableStatistics = getBasicStatisticsForTable(onHive(), tableName);
+                assertThatStatisticsAreNotPresent(tableStatistics);
+            }
```

Comment:
```
if (getHiveVersionMajor() == 3) {
  assertBasicStats();
}
```
?

Here and below

Reply:
Why `==` ?

Reply:
in commit message you said that hive 3 has special behaviour that basis stats are present even without analyze. So I suggested to write a test for this behaviour.

Reply:
we're not relying on this behavior, it doesn't matter to Presto